tonic_textual/redact_api.py

```diff
@@ -221,6 +224,7 @@ def redact(
         random_seed: Optional[int] = None,
         label_block_lists: Optional[Dict[str, List[str]]] = None,
         label_allow_lists: Optional[Dict[str, List[str]]] = None,
+        record_options: Optional[RecordApiRequestOptions] = default_record_options
```

Comment:
```suggestion
        record_options: RecordApiRequestOptions] = default_record_options
```

If `record_options` is `None`, then line 301 throws `AttributeError: 'NoneType' object has no attribute 'record'`.  The `record_options` parameter does not need to be optional because it has the default value that's set.